zerver/models.py

```diff
@@ -292,7 +290,13 @@ class Realm(models.Model):
     UPLOAD_QUOTA_STANDARD = 50
     upload_quota_gb = models.IntegerField(null=True)  # type: Optional[int]
 
-    video_chat_provider = models.CharField(default=u"Jitsi", max_length=MAX_VIDEO_CHAT_PROVIDER_LENGTH)
+    JITSI_MEET = u"Jitsi"
+    GOOGLE_HANGOUTS = u"Google Hangouts"
+    ZOOM = u"Zoom"
```

Comment:
Let's call these variables e.g. `VIDEO_CHAT_JITSI_MEET`; we generally use that scoping pattern for variable names for enums in our models, for clarity.